src/main/java/com/zepben/cimbend/database/sqlite/writers/NetworkCIMWriter.kt

```diff
@@ -585,6 +585,7 @@ class NetworkCIMWriter(databaseTables: DatabaseTables) : BaseCIMWriter(databaseT
         val insert = databaseTables.getInsert(TablePowerTransformers::class.java)
 
         insert.setNullableString(table.VECTOR_GROUP.queryIndex, powerTransformer.vectorGroup.name)
+        insert.setDouble(table.TRANSFORMER_UTILISATION.queryIndex, powerTransformer.transformerUtilisation)
```

Comment:
There should also be a test for read/writing this. There should be a `fillFields` method (or similar) for each class that is used to make sure all fields actually read/write to the correct propterties on the CIM classes.